code/datums/trading/goods.dm

```diff
@@ -380,6 +372,7 @@ Sells devices, odds and ends, and medical stuff
 		"hail_deny"         = "ORIGIN no longer wants to speak to you.",
 		"trade_complete"    = "Good mining and avoid the holes!",
 		"trade_blacklist"   = "I don't want this thing.",
+		"what_want"			= "You got any leftover materials? Specifically",
```

Comment:
Wrong spacing between "what_want" and the text. Use space

Reply:
Adjusted and pushed. Resolving.